sdk/cosmos/test-resources.json

```diff
@@ -79,6 +81,19 @@
         "capabilities": [],
         "ipRules": []
       }
+    },
```

Comment:
@yiliuTo - We don't need this extra entry in test-resources.json file. We can just re-use this file as it is. 
The way it works is, for each matrix entry in tests.yml, it will create a new account using this test-resources.json, so you just need to add the new matrix entry for spring in tests.yml which I see you have already added. 

Though one thing which needs to be changed will be how we handle the output of this test-resources.json file in spring integration tests, but don't worry about it in this PR, I will take care of it once this PR goes in.